graph-backup-job/overlays/aws-prod/kustomization.yaml

```diff
@@ -23,26 +22,8 @@ patches:
       annotationSelector: "operation=chat-notification"
   - patch: |-
       - op: add
-        path: "/spec/jobTemplate/spec/template/spec/containers/0/env/-1"
-        value: {"name": "THOTH_DATA_DUMP_PATH", "value": "/var/lib/pgsql/pg_dump.sql"}
-    target:
-      group: batch
-      version: v1beta1
-      kind: CronJob
-      name: graph-backup
-  - patch: |-
-      - op: add
-        path: "/spec/jobTemplate/spec/template/spec/containers/0/volumeMounts/-1"
-        value: {"name": "postgresql-data","mountPath": "/var/lib/pgsql"}
-    target:
-      group: batch
-      version: v1beta1
-      kind: CronJob
-      name: graph-backup
-  - patch: |-
-      - op: add
-        path: "/spec/jobTemplate/spec/template/spec/volumes/-1"
-        value: {"name": "postgresql-data","persistentVolumeClaim":{"claimName": "dbbackup"}}
+        path: "/spec/jobTemplate/spec/template/spec/containers/0/resources"
+        value: {"requests": {"memory":"1Gi", "cpu": "1", "ephemeral-storage": "20Gi"}, "limits": {"memory":"1Gi", "cpu": "1", "ephemeral-storage": "20Gi"}}
```

Comment:
```suggestion
        path: "/spec/jobTemplate/spec/template/spec/containers/0/resources/requests/ephemeral-storage"
        value: "20Gi"
```

Reply:
```suggestion
      - op: add
        path: "/spec/jobTemplate/spec/template/spec/containers/0/resources/limits/ephemeral-storage"
        value: "20Gi"
```

Reply:
I would only add the ephemeral in the patch (same for other environment).
Or maybe we can add 20Gi as the base and only override test at 1Gi ?

Side note: Do we want to add the limit on ephemeral-storage ? The idea is to be alerted immediately if db dump starts to go over 20Gi ?

Reply:
(not sure if my second sugg applies cleanly on top on the other, but the idea : two add ops specifying the value only at the correct path.

Reply:
Ack, i have made the changes as you suggested.
The limits is something requested by cluster team so fit with the namespace limits.

about we being alerted , the exceed of those limit would cause failure in system, so we should be notified.